README.md

```diff
@@ -135,7 +135,7 @@ I would like to thank to my coding partners.
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 
```

Comment:
![image](https://github.com/ibugithub/math-Magician/assets/93039359/5bc93c5e-8cfd-4e46-95d9-c524970f6fb5)

- [ ] Kindly fix the highlighted links in the images above because these links are broken 😨 Kindly fix these links to make your readme descriptive and professional 👍